lib/elixir/lib/kernel/cli.ex

```diff
@@ -16,6 +16,8 @@ defmodule Kernel.CLI do
     profile: nil
   }
 
+  @standalone_opts ["-h", "--help", "--short-version"]
```

Comment:
```suggestion
  @standalone_opts ["-h", "--help", "-v", "--version", "--short-version"]
```

Reply:
You were too fast, I changed my mind on not making -v and --version not standalone. Sorry. :)

Reply:
Btw, please check it doesn't break `iex --version`. It may be we pass some implicit `iex` flags.

Reply:
Hahah, alright then :D